src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -311,7 +291,13 @@ public static SlimefunItem getByID(String id) {
 	}
 
 	public static SlimefunItem getByItem(ItemStack item) {
-		if (item == null) return null;		
+		if (item == null) return null;
+
+		if (item.getItemMeta() != null) {
```

Comment:
You should do hasItemMeta() instead of a null-check

Reply:
hasItemMeta literally is just a null check and this stops IDEs from complaining that it may be null.